### ISSUE TITLE
Pretty printed file has a line with thousand of characters

### ISSUE THREAD

**Issue by jasonLaster** [RecordReplay/devtools#4420]
STR:
1. go to cord.com
2. click Try
3. record example

https://app.replay.io/recording/4b62a997-4cbc-4284-8965-0ded391096dc?point=183677501433904126717086820059840909&time=70073.54936562151&hasFrames=true#

original replay: https://app.replay.io/recording/7c765bd6-50e1-40ab-83ce-bc2c6ee04eb6

**Comment by hbenl** [comment]
This seems to be a bug in `js-beautify` with the `e4x` option that we recently enabled.
To reproduce:
```
npm i -g js-beautify
curl https://app.cord.com/1.82.0/sidebar-eed1df1bbf4c2ba31c69ae11a7955e16.js >sidebar.js
js-beautify -X sidebar.js >beautified.js
```

Then open `beautified.js` in an editor and search for "addAnnotation"...

**Comment by hbenl** [comment]
I created an issue for this: beautify-web/js-beautify#1962

**Comment by hbenl** [comment]
Fixed by RecordReplay/backend#3532
